cmd/openshift-install/create.go

```diff
@@ -131,6 +131,7 @@ var (
 					if err2 := logClusterOperatorConditions(ctx, config); err2 != nil {
 						logrus.Error("Attempted to gather ClusterOperator status after installation failure: ", err2)
 					}
+					// TODO this error would be better rendered with newlines intact instead of quoted.
```

Comment:
This error is not quoted for interactive-terminal users. So this TODO is not accurate.